src/histogram_equalization.jl

```diff
@@ -0,0 +1,217 @@
+function to_uint8(img)
```

Comment:
Is this doing the same as `_convert_to_255_matrix()`? If so, maybe only one them is needed?

Reply:
They are not quite the same thing. This one takes a matrix of floats in the range [0, infinity) and converts its members to UInt8 (0-255). `convert_to_255_matrix` takes a matrix of Gray{Float} with values in [0, 1] and converts them to UInt8. Their target domain is the same but the input images are different in data type.